libs/portals/admin/service-desk/src/hooks/useNationalId.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react'
+
+interface UseNationalId {
+  nationalId: string
+  handleInputChange: (value: string) => void
+  isValid: boolean
+  formatNationalId: (value: string) => string
+}
+
+export const useNationalId = (): UseNationalId => {
```

Comment:
After getting better context I see that this hook is redundant. You could just use the utility method and remove this hook. Unless you where planning of using it?